src/core/layout/grid/grid.js

```diff
@@ -33,6 +33,7 @@ export class GelGrid extends Phaser.GameObjects.Container {
     }
 
     addGridCells(theme) {
+        this.page = this.getCellPage(theme, this._config.showChoice);
```

Comment:
Any reason why you aren't using the `transientData.choice` value here that was getting saved anyway?

Reply:
I've been over it again and while using `choice` does technically work, we would be saying that choice transient data does not necessarily reflect the choice which the player made.
 
If we are happy saying that then fine, I think it slightly muddies whatever the game does around the selected choice but any game changing this should be aware of what the value means anyway.